blocks/large-promo-block/features/large-promo/default.jsx

```diff
@@ -225,9 +237,7 @@ const LargePromoItem = ({ customFields, arcSite }) => {
 					</MediaItem>
 				) : null}
 				<Stack className={`${BLOCK_CLASS_NAME}__text`}>
-					{showOverline && (url || text) ? (
-						<Overline href={url ? formatURL(url) : null}>{text}</Overline>
-					) : null}
+					{showOverline ? <Overline href={url}>{text}</Overline> : null}
```

Comment:
Was this lost or not necessary?
```suggestion
					{showOverline ? <Overline href={formatURL(url)}>{text}</Overline> : null}
```

Reply:
@nschubach Thanks, removed formatURL() in [3b044d8](https://github.com/WPMedia/arc-themes-blocks/pull/1462/commits/3b044d874ca3e8201b6c45351ab8f9d750f61534)

Reply:
@malavika37  be sure to add this one back in.

Reply:
@nschubach added it back in [beaaa9c](https://github.com/WPMedia/arc-themes-blocks/pull/1462/commits/beaaa9ce26ab3304976068d9261376934f2b47b5)

Reply:
@malavika37 not this specific case.